### DISCUSSION TITLE
Request: Rails notifications with Action Cable and notifications JS library

### DISCUSSION THREAD

**Discussion by preist** [jonhue/notifications-rails#206] [Ideas]
This is a feature request:

Have an example that implements Rails notifications with Action Cable and a [toastr](https://github.com/CodeSeven/toastr) style notification JavaScript library. This would make the library more popular and help users ramp up to use it.

**Comment by jonhue** [comment]
I originally intended to write a frontend for ActionCable for this gem, but never got around to actually doing it. Right now, the only frontends maintained by me are for ActionMailer and OneSignal. If you want to build such a frontend, I'd be glad to answer your questions if you have any.
